src/setupProxy.js

```diff
@@ -1,27 +1,33 @@
 const { createProxyMiddleware } = require('http-proxy-middleware')
 
+const { PUBLIC_URL = '' } = process.env
+
 module.exports = (app) => {
   app.use(
-    '/api/',
+    `${PUBLIC_URL}/api/`,
     createProxyMiddleware({
       target: 'https://localhost:28183',
       changeOrigin: true,
       secure: false,
       ws: false,
       onProxyReq: (proxyReq, req, res) => {
+        proxyReq.path = proxyReq.path.replace(PUBLIC_URL, '')
         if (req.headers['x-jm-authorization']) {
           proxyReq.setHeader('Authorization', req.headers['x-jm-authorization'])
         }
       },
     })
   )
   app.use(
-    '/ws/',
+    `${PUBLIC_URL}/ws/`,
     createProxyMiddleware({
       target: 'wss://localhost:28183',
       changeOrigin: true,
       secure: false,
       ws: true,
+      onProxyReq: (proxyReq, req, res) => {
+        proxyReq.path = proxyReq.path.replace(PUBLIC_URL, '')
```

Comment:
Would using `pathRewrite` for this be more natural?
A [custom function](https://github.com/chimurai/http-proxy-middleware/blob/master/recipes/pathRewrite.md#custom-rewrite-function) can be defined which we can be reused for every existing and future route.

What do you think?

Reply:
Yes, I think it can be cleaned up – for now this is just what I came up with to quickly test it. I'm not sure if we'd want to keep the contents of my last commit, as it is  intended for testing purposes only.

Reply:
I can rebase and clean it up once the websocket PR is merged or simply remove the last commit once you had a chance to verify it. Depends on whether or not we want to introduce the nginx into the regtest setup - it is only needed for testing this use case and possibly providing a reference for the integration, but we also got the docker repo for that.